packages/ember-runtime/lib/mixins/promise_proxy.js

```diff
@@ -38,7 +38,7 @@ function tap(proxy, promise) {
   let ObjectPromiseProxy = Ember.ObjectProxy.extend(Ember.PromiseProxyMixin);
 
   let proxy = ObjectPromiseProxy.create({
-    promise: $.getJSON('/some/remote/data.json')
+    promise: Ember.RSVP.cast($.getJSON('/some/remote/data.json'))
```

Comment:
`Ember.RSVP.cast` is actually undocumented and [in the source](https://github.com/tildeio/rsvp.js/blob/5727073c53c207bb687b82a1683eb3c1b332960b/lib/rsvp.js#L26) seems to be an alias to `resolve` Maybe _resolve_ is a better choice since it is documented and used in the guides.